frontend/src/scenes/experiments/Experiments.tsx

```diff
@@ -40,37 +41,60 @@ export function Experiments(): JSX.Element {
                 )
             },
         },
+        createdByColumn<Experiment>() as LemonTableColumn<Experiment, keyof Experiment | undefined>,
+        createdAtColumn<Experiment>() as LemonTableColumn<Experiment, keyof Experiment | undefined>,
         {
-            title: 'Start Date',
-            dataIndex: 'start_date',
-            render: function RenderStartDate(_, experiment: Experiment) {
-                return experiment.start_date ? (
-                    <div style={{ whiteSpace: 'nowrap' }}>
-                        <TZLabel time={experiment.start_date} />
+            title: 'Duration',
+            render: function Render(_, experiment: Experiment) {
+                return (
+                    <div>
+                        {`${
+                            experiment.end_date
+                                ? dayjs(experiment.start_date).diff(dayjs(experiment.end_date), 'day')
+                                : 0
+                        }`}{' '}
+                        days
                     </div>
-                ) : (
-                    <span style={{ color: 'var(--muted)' }}>Draft</span>
                 )
             },
-            sorter: (a, b) => (new Date(a.start_date || 0) > new Date(b.start_date || 0) ? 1 : -1),
         },
-        createdByColumn<Experiment>() as LemonTableColumn<Experiment, keyof Experiment | undefined>,
-        createdAtColumn<Experiment>() as LemonTableColumn<Experiment, keyof Experiment | undefined>,
+        {
+            title: 'Status',
+            render: function Render(_, experiment: Experiment) {
+                const statusColors = { active: 'green', draft: 'red', complete: 'purple' }
```

Comment:
@clarkus I know in the mocks we set the "draft" tag to gray but there's no preset gray color for it, so I wanted to avoid writing extra conditionals just to check for that, do we feel strongly about it being gray or can we make use of another preset color? 😁

Reply:
Think this should work?

```suggestion
                const statusColors = { active: 'green', draft: '#108ee9', complete: 'purple' }
```

Reply:
well yes but then we have to change the font color for just the word "DRAFT" itself because custom colors default to a white text (which is hard to see on the gray background)

Reply:
Color isn't super meaningful here - it's mostly to add another means of distinguishing items. Draft should probably look the least "active". Could be another color, but I'd avoid greends, blues, reds, etc. In the designs the pill background color is #EAECF1. The text color is our muted alt secondary text (#747EA1).

Reply:
<img width="395" alt="Screen Shot 2021-12-15 at 11 03 56 AM" src="https://user-images.githubusercontent.com/25164963/146221323-6e774303-978b-43f6-a029-2c7efb4de811.png">

anything from this palette look good? if not going with the gray is nbd

Reply:
oh wait! there's a default looking one here 
<img width="88" alt="Screen Shot 2021-12-15 at 11 05 38 AM" src="https://user-images.githubusercontent.com/25164963/146221581-63768e29-c66d-4445-9836-fe1abe905a0d.png">

Reply:
<img width="137" alt="Screen Shot 2021-12-15 at 11 07 23 AM" src="https://user-images.githubusercontent.com/25164963/146221892-bf33ba8a-2f03-4880-8657-c378042113ec.png">